gulpfile.js

```diff
@@ -67,13 +66,30 @@ gulp.task('copy-content-assets', () => {
       // they belong to.
       .pipe(
         rename((assetPath) => {
-          const leftPart = assetPath.dirname.split('/')[1];
-          // Let certain path's images to pass through.
+          const parts = assetPath.dirname.split('/');
+          // Landing pages should keep their assets.
+          // e.g. en/vitals, en/about
+          if (parts.length === 2) {
```

Comment:
can you make this <= 2? I don't think any images will ever appear at the actual top-level but the rest of the method mostly works by chance (`subdir` will be `undefined`, `splice` will do nothing).